docs/reference/query_types/parameters/common/query/sort/body.rst.inc

```diff
@@ -18,6 +18,15 @@ Following sort clauses are available through string definition:
     :depth: 1
     :local:
 
+Parent sort clausses
+~~~~~~~~~~~~~~~~~~~~
+
+String ``parent`` enables sorting by settings on parent location in administration.
+
+.. code-block:: yaml
+
+sort: parent
```

Comment:
Missing indent, it's not formatting properly.

Reply:
@pspanja This change has been reverted.